src/arithmetic/entity_funcs/entity_funcs.c

```diff
@@ -130,6 +131,12 @@ SIValue AR_PROPERTY(SIValue *argv, int argc) {
 	// return NULL for missing graph entity
 	if(SI_TYPE(argv[0]) == T_NULL) return SI_NullVal();
 
+	if(SI_TYPE(argv[1]) != T_STRING) {
```

Comment:
Please add a short explanation why we're still checking `argv[1]` given our invocation verification,
`AR_SUBSCRIPT` ...